internal/runtime/interface.go

```diff
@@ -289,11 +289,8 @@ type GroundedSecret struct {
 }
 
 type ContainerRunOpts struct {
-	WorkingDir         string
 	Image              oci.ImageID
-	Command            []string
-	Args               []string
-	Env                []*schema.BinaryConfig_EnvEntry
+	BinaryConfig       *schema.BinaryConfig
```

Comment:
I mentioned this above on `Invocation` but it's the same problem here -- it doesn't feel like this is going in the right direction. Previously ContainerRunOpts' interface was fairly well defined. Although it was already a challenge that it re-used BinaryConfig_EnvEntry, which has multiple meanings.

Embedding BinaryConfig means that future updates to BinaryConfig will end up being part of the runtime interface, and that's not quite right, as we're mixing two different APIs. I think it would be best to keep the fields that a runtime interface can act on explicitly declared, so Args, Command, Env etc separate as before.